src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -22,19 +24,33 @@ const items = [
 
 const ProfileSection: React.FC<Props> = ({
   email,
+  appTheme,
   hasPassword,
   updatePassword,
   updateLanguage,
+  updateTheme,
 }) => {
   const intl = useIntl();
   const [isOpen, setIsOpen] = useState(false);
   const [currentLang, setCurrentLang] = useState<string>();
-
+  const [currentThemeValue, setCurrentThemeValue] = useState<string>();
+  const [currentThemeLabel, setCurrentThemeLabel] = useState<string>();
+  const themeItems = [
+    { key: "DARK", label: intl.formatMessage({ defaultMessage: "Dark theme" }), icon: "moon" },
+    { key: "LIGHT", label: intl.formatMessage({ defaultMessage: "Light theme" }), icon: "sun" },
+  ];
   useEffect(() => {
     const lang = items.find(item => item.key === intl.locale);
     setCurrentLang(lang?.label);
   }, [intl.locale]);
 
+  useEffect(() => {
+    const appThemeValue = appTheme ?? "DARK";
+    setCurrentThemeValue(appThemeValue);
```

Comment:
Sorry I think you misunderstood my suggestion (unless my suggestion didn't work. If It didn't work please leave a comment before resolving the comment so I and others know).
You can move the `appTheme ?? "DARK"` into the useState declaration and here just have `setCurrentThemeValue(appTheme)`.
That way the currentThemeValue starts with a value on render and the useEffect is just updating if/when appTheme is defined.

Reply:
I need this variable for later usages, so I made the declaration like this.